### ISSUE TITLE
 v4.1.1 Recent Queries not work: ReferenceError: moment is not defined

### ISSUE THREAD

**Issue by polygontwist** [pi-hole/pi-hole#2562]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*.
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**

`{7}`

---
**Expected behaviour:**

`{show query log}`

**Actual behaviour:**

`{noting; Console wrote: ReferenceError: moment is not defined queries.js:295:101 }`

**Steps to reproduce:**

`{Detailed steps of how we can reproduce this}`

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**

`{qkl3mg6itn}`

**Troubleshooting undertaken, and/or other relevant information:**

`{Login, clikc "Query Log"}`

**Comment by Mcat12** [comment]
This is a web issue, not a core issue. Have you tried clearing your browser's cache? You may have some old web interface files cached.

**Comment by polygontwist** [comment]
yes the cache ist cleaned - I have tested it on different browsers and devices. (Under Safari on iOS9, the interface generally does not work.). Core or interface - as a user, it belongs to me. sorry. If I can not see the list it is broken.

**Comment by Mcat12** [comment]
Do you see any issues in the web interface, besides the message in the console?

**Comment by polygontwist** [comment]
clik: "Long term data" and sub-points (2 Errors): 
ReferenceError: moment is not defined[Weitere Informationen] daterangepicker.js:32:5
http://192.168.0.100/admin/scripts/vendor/daterangepicker.js:32:5
http://192.168.0.100/admin/scripts/vendor/daterangepicker.js:9:2
ReferenceError: moment is not defined[Weitere Informationen] db_graph.js:12:5
http://192.168.0.100/admin/scripts/pi-hole/js/db_graph.js:12:5

the other has no error message in the log.

**Comment by Mcat12** [comment]
Besides the errors in the console, do you see anything wrong in the web interface, such as incorrect data?

**Comment by polygontwist** [comment]
I do not see any data. The dashboard works.
![grafik](https://user-images.githubusercontent.com/3751286/50496626-830f3d00-0a31-11e9-993d-51e9ea2ed8dd.png)

**Comment by polygontwist** [comment]
in which format must the date be? I entered 29.12.2018, 2018-12-28, 2018.12.29 - nothing happens. There is also no console output. With "Query Log" the error also comes.

Firefox:
![grafik](https://user-images.githubusercontent.com/3751286/50538332-2feed480-0b6e-11e9-91ce-f9099a2df820.png)

Chrome:
![grafik](https://user-images.githubusercontent.com/3751286/50538353-a986c280-0b6e-11e9-930b-127c59f75091.png)

**Comment by DL6ER** [comment]
Are these recent versions of the two browsers? Which operating system?

Does the file `/var/www/html/admin/scripts/vendor/moment.min.js` exist on your Pi-hole? It should be about 60KB large and have (at least) the permissions `-rw-r--r--`.
